packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 9.0.0
+
+* **Breaking Change** Updates `DartOptions` to be immutable and adds const to the constructor.
+* **Breaking Change** Reverts `final` changes to Flutter Api classes.
```

Comment:
I don't think this one is breaking? Is there code that wouldn't compile due to something no longer being final?

Reply:
I guess it's more of an un-breaking change